west.yml

```diff
@@ -139,7 +139,7 @@ manifest:
       groups:
         - hal
     - name: hal_ti
-      revision: 5e7d5cd584047699c9fd279923120cb25ba3dda7
+      revision: cd40806ee15f4f8f73e1561d3c0e9d99444ea8f0
```

Comment:
revision should be `pull/45/head` here until the `hal_ti` PR is merged, but that cannot be merged until this PR is all green. After the `hal_ti` PR is merged, this can be changed to the new HEAD commit in `hal_ti`

Reply:
@jadonk - kind of a weird little dance with module dependencies. Sorry for the delay. Documented here:

https://docs.zephyrproject.org/latest/develop/modules.html#submitting-changes-to-modules

Reply:
I'm not sure how to mark it DNM, but I made the west.yml change to `pull/45/head`.

Reply:
I think I've handled these requests. For the typedefs, I hope you can make a suggestion as this must be a common problem.